mmbert.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: "mmBERT: A Modern Multilingual Encoder with Annealed Language Learning" 
+title: "mmBERT: ModernBERT goes Multilingual" 
```

Comment:
We can diverge from the paper title, e.g. see the [ModernBERT blogpost](https://huggingface.co/blog/modernbert) (`Finally, a Replacement for BERT` VS `Smarter, Better, Faster, Longer: A Modern Bidirectional Encoder for Fast, Memory Efficient, and Long Context Finetuning and Inference`). 

I think we can go for a snappy title that shows the model family name while immediately explaining what the model is and how it's an improvement over prior work, e.g.: `mmBERT: ModernBERT goes Multilingual`.